### ISSUE TITLE
Option Chain for MCX

### ISSUE THREAD

**Issue by ChaitanyaKS2001** [Shoonya-Dev/ShoonyaApi-py#192]
Is it possible to use api.get_option_chain() with MCX as exchange? 
If yes, please provide a sample example for CRUDEOIL and NATURALGAS.
When I'm using it, nothing is being returned.

**Comment by Dappsnode** [comment]
@ChaitanyaKS2001 Yes, it's possible to use `api.get_option_chain()` with MCX as the exchange, but it depends on the specific API service you're using. Different financial data providers have different implementations and support for exchanges like MCX.If you're using a specific API service, make sure to check their documentation for details on how to fetch option chains for commodities like CRUDEOIL and NATURALGAS. Below is a general example of how you might use such a function

**Comment by ChaitanyaKS2001** [comment]
@Dappsnode you gave a generalized answer, if possible give some examples by passing values as arguments in the function `api.get_option_chain()`

**Comment by abhisheksoni27** [comment]
??

**Comment by MuskanPujara** [comment]
Yeah I have used it you can use 
api.get_option_chain('MCX','CRUDEOILMCX16DEC3800CE,3800,2)
